projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-dimensions.ts

```diff
@@ -94,6 +100,8 @@ export class IgxPivotDateDimension implements IPivotDimension {
             return;
         }
 
+        this.dataType = inBaseDimension.dataType;
```

Comment:
Shouldn't the child dimensions below also apply a data type? Otherwise I assume they won't be sorted as dates but as strings as well.